pyvista/core/pointset.py

```diff
@@ -478,6 +503,28 @@ def n_open_edges(self):
         alg.Update()
         return alg.GetOutput().GetNumberOfCells()
 
+    def remove_faces(self, ind):
```

Comment:
I'd remove this method in favor of Zen's

> There should be one-- and preferably only one --obvious way to do it.

I think `remove_cells` is understandable in the context of a surface mesh as well.

Reply:
Agreed.  This also means that we should remove `faces`.  This will be a significant refactor.